doc/api/crypto.md

```diff
@@ -3432,6 +3474,52 @@ const n = crypto.randomInt(1, 7);
 console.log(`The dice rolled: ${n}`);
 ```
 
+### `crypto.randomPrime(size[, options[, callback]])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `size` {number} The size (in bytes) of the prime to generate.
+* `options` {Object}
+  * `add` {ArrayBuffer|SharedArrayBuffer|TypedArray|Buffer|DataView}
+  * `rem` {ArrayBuffer|SharedArrayBuffer|TypedArray|Buffer|DataView}
+  * `safe` {boolean}
+* `callback` {Function}
+  * `err` {Error}
+  * `prime` {ArrayBuffer}
```

Comment:
I always felt that it was strange that WebCrypto returns `ArrayBuffers`, not `Uint8Arrays`. What's the motivation here?

Reply:
The biggest reason, to be honest, is that [`createDiffieHellman`](crypto_crypto_creatediffiehellman_prime_primeencoding_generator_generatorencoding) expects the prime to be a string, TypedArray, or ArrayBuffer. Having randomPrime return an ArrayBuffer makes the output directly usable by the existing api as opposed to returning a bigint. 

This approach also makes it trivial to use the primes as inputs to other functions (raw key material for instance). Bigints may be more JavaScripty but ArrayBuffer is more flexible with regards to existing APIs. 

Once we bigint enable the existing APIs then I agree it would be an option.

Reply:
I understand the motivation, but to me, it seems like we are only making things worse by intentionally steering away from JS types. Sure, we could patch in an option later to return a `BigInt` instead, but the default behavior would always be "the old way."

Reply:
Well, it's just going to be an ergonomic headache for this API to support returning bigint's by default if they can't be used anywhere else without being transformed into an ArrayBuffer or Uint8Array. I can certainly add the option for generating a bigint now but I'd be -1 on making it the default at this time.

Reply:
Ok, I've updated to support bigints via an option...

```js
const prime = crypto.randomPrime(1024, { bigint: true });
console.log(typeof prime); // bigint
``` 

It's also been updated to bits instead of bytes.